cmd/blockchaincmd/deploy.go

```diff
@@ -1174,6 +1176,22 @@ func deployBlockchain(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func setBootstrapValidatorValidationID(avaGoBootstrapValidators []*txs.ConvertSubnetToL1Validator, bootstrapValidators []models.SubnetValidator, subnetID ids.ID) error {
+	for index, avagoValidator := range avaGoBootstrapValidators {
```

Comment:
you can use utils.MapWithError for this.
or utils.Map if error can be ignored.

imho this error doesn't need to be raised up and just skipped as we try to match one thing to another so any invalid nodeID can be ignored as it would not match

Reply:
if you don't need err, you should be able to use utils.Map
just a nit, pls feel free to ignore it.